src/main/java/com/exasol/bucketfs/ReadEnabledBucket.java

```diff
@@ -240,6 +274,10 @@ public String toString() {
         return (this.bucketFsName == null ? (this.port + ":") : (this.bucketFsName + "/")) + this.bucketName;
     }
 
+    /**
+     * Returns a builder.
```

Comment:
Please reformat all files using the eclipse formatter.

Reply:
done!